modules/apps/commerce/commerce-frontend-taglib/src/main/resources/META-INF/resources/header/page.jsp

```diff
@@ -134,14 +134,9 @@ String myWorkflowTasksPortletNamespace = PortalUtil.getPortletNamespace(PortletK
 
 					<div class="col-auto d-lg-none">
 						<clay:button
-							data='<%=
-								HashMapBuilder.<String, String>put(
-									"target", "#navbarNavAltMarkup"
-								).put(
-									"toggle", "collapse"
-								).build()
-							%>'
-							elementClasses="navbar-toggler p-3"
+							data-target="#navbarNavAltMarkup"
+							data-toggle="liferay-collapse"
+							elementClasses="navbar-toggler"
```

Comment:
Small nit... `cssClass` instead of `elementClasses` whenever you can, since we're deprecating that one.

Reply:
Totally right, thx!